src/docs/grid-aware-websites/tutorials/grid-aware-tutorial-cloudflare-workers.md

```diff
@@ -71,7 +71,7 @@ The `src/index.js` file contains the Worker code that we will modify to add grid
 
 ### Setting routes
 
-Before we start writing code, we'll first configure our worker to run on the route we want it to apply to. We want to apply this Worker to the `/tools/grid-aware-websites/` path on the `thegreenwebfoundation.org` domain. To do that, we include the following configuration inside of the `wrangler.json` file. You should replace the `pattern`, and `zone_name` with your own desired route.
+Before we start writing code, we'll first configure our worker to run on the route we want it to apply to. We want to apply this Worker to the `/tools/grid-aware-websites/` path on the `thegreenwebfoundation.org` domain. To do that, we include the following configuration inside of the `wrangler.jsonc` file. You should replace the `pattern`, and `zone_name` with your own desired route.
```

Comment:
When using the `create` command back in December, for some reason I got a `wrangler.toml` file. This time I got a `wrangler.jsonc`. The [routes docs](https://developers.cloudflare.com/workers/configuration/routing/routes/) only mention those two formats, no `wrangler.json`.

I assume they might have changed this recently?